### PR TITLE
add example in cli for --cwd option

### PR DESCRIPTION
**Pull request by guillaumevincent**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -97,6 +96,14 @@ $ cpy --help
...
 
   <source> can contain globs if quoted
+
+  Examples
+    $ cpy 'src/*.png' '!src/goat.png' dist
+    copy all .png files in src folder into dist except src/goat.png
+
+    $ cpy '**/*.html' '../dist/' --cwd=src --parents
+    copy all .html files inside src folder into dist and keep path structure
+
```
Comment (schnittstabil) [inline readme.md]: Please remove the last empty line.

File: cli.js [hunk 2]
```diff
@@ -8,16 +8,20 @@ var cli = meow({
...
 		'',
-		'<source> can contain globs if quoted'
+		'<source> can contain globs if quoted',
+		'',
+		'Examples',
+		'  $ cpy \'src/*.png\' \'!src/goat.png\' dist',
+		'  copy all .png files in src folder into dist except src/goat.png',
+		'',
+		'  $ cpy \'**/*.html\' \'../dist/\' --cwd=src --parents',
+		'  copy all .html files inside src folder into dist and keep path structure'
```
Comment (schnittstabil) [inline cli.js]: please change `keep` to `preseve`, because:

> --parents            Preseve path structure'

File: readme.md [hunk 3]
```diff
@@ -97,6 +96,14 @@ $ cpy --help
     --rename=<filename>  Rename all <source> filenames to <filename>
 
   <source> can contain globs if quoted
+
+  Examples
+    $ cpy 'src/*.png' '!src/goat.png' dist
+    copy all .png files in src folder into dist except src/goat.png
+
+    $ cpy '**/*.html' '../dist/' --cwd=src --parents
+    copy all .html files inside src folder into dist and keep path structure
```
Comment (schnittstabil) [inline readme.md]: same here: s/keep/preseve/

File: cli.js (hunk 2 above)
Comment (schnittstabil) [inline cli.js]: Oh, a typo :smile:

**Comment by guillaumevincent** [comment]
@schnittstabil Preserve not Preseve ?

**Comment by schnittstabil** [comment]
@guillaumevincent Many thanks :+1:

**Comment by sindresorhus** [comment]
Thank you @guillaumevincent :)

**Pull request merged**